examples/material_gallery/lib/slider_demo.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'widget_demo.dart';
+
```

Comment:
why not merge the SliderExample and the SliderDemo?
At least make SliderDemo Stateless if you want to keep it.